frontend/src/App.js

```diff
@@ -24,6 +24,7 @@ export default function App() {
           <Switch>
             <Route path="/mapview">
               <div className="m-4 p-3">
+                <title>Térkép nézet</title>
```

Comment:
Az egész appon belüli részt érdemes a Providerbe rakni, a title-t pedig Helmet-be. RTFM.